src/delta_lake/delta_service.py

```diff
@@ -122,3 +126,43 @@ def sample_delta_table(
         raise SparkOperationError(
             f"Failed to sample rows from {full_table_name}: {str(e)}"
         )
+
+
+def query_delta_table(
```

Comment:
Is the intent here to allow arbitrary select queries? If so, what can this do that the prior method can't do assuming the limit was allowed to be infinite, and do we need to do that?

Reply:
Join of different tables.

Reply:
Should each of the target tables be inputs in that case?

Reply:
👍 I think it's better that we just remove them.